README.md

```diff
@@ -39,46 +40,62 @@ Concretely, it's a *typed* content address: a tuple of `(content-type, content-a
 
 Current version: CIDv1
 
-A CIDv1 has four parts:
+The atomic CIDv1 is a **binary** format composed from `unsigned_varint`s that encode values from the , each of which :
 
 ```text
-<cidv1> ::= <mb><multicodec-cidv1><mc><mh>
+<cidv1> ::= <CIDv1-multicodec><content-multicodec><multihash>
 # or, expanded:
-<cidv1> ::= <multibase-prefix><multicodec-cidv1><multicodec-content-type><multihash-content-address>
+<cidv1> ::= <`0x01`, the multicodec literal for `CIDv1`><multicodec bytecode signaling content type of the data being addressed><multihash of that data>
```

Comment:
Oh, I see you're explaining it here

```suggestion
<cidv1> ::= <`0x01`, the multicodec code for `CIDv1`><multicodec code signalling content type of the data being addressed><multihash of that data>
```

I'm mildly uncomfortable with "content type" but maybe that'll have to do without diverging into having to explain the vagaries of IPLD here.

Reply:
You make a great point that maybe IETF won't love the "content type" analogy, given that the IANA "content types" registry is the canonical example of an IANA registry that the whole web uses for everything, and we should be avoiding the interpretation that all this is just the an alternate stack for a micro-web.  

My addled brain reached for the nearest analogue but what I was *really* trying to do was explain a CID as a turducken or a matrushka: a multibase (or a multibase-NUL if being exported to CBOR-parseable stream) wrapping a CIDv1-tag wrapping any other valid muticodec tag wrapping an innermost payload ("content" if you've parsed all the preceding tags correctly).  But the word "content" is overdetermined here so I should keep trying.

```suggestion
<cidv1> ::= <`0x01`, the multicodec code for `CIDv1`><another multicodec code that identifies the type of the following digest><multihash digest of any unit of data>
```

Or your version if I've overstretched the definition to avoid using the word "content" 😅